internal/pkg/admission/policy-server-configmap.go

```diff
@@ -34,38 +25,55 @@ type policyServerConfigEntry struct {
 func (r *Reconciler) reconcilePolicyServerConfigMap(
 	ctx context.Context,
 	policyServer *policiesv1alpha2.PolicyServer,
-	operation policyServerConfigMapOperation,
+	clusterAdminPolicies *policiesv1alpha2.ClusterAdmissionPolicyList,
```

Comment:
```suggestion
	clusterAdmissionPolicies *policiesv1alpha2.ClusterAdmissionPolicyList,
```

Typo? And instances tied to this one.

Reply:
yes it is a typo. I just made a commit. Thanks!